### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by crdanielbusch**
Work in progress

## Checklist

Please confirm that this pull request has done the following:

- [ ] Tests added
- [ ] Documentation added (where applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,39 +146,75 @@ To calculate emissions for all layers, run `omPrior.py` with a start and end dat
...
+To carry out the steps described above in a Docker container, first build the Docker image with
+
+```
+make build
+```
+Then run the commands to download the input data in the docker container
+
+```
+docker run --rm -v </your/path/to/openmethane-prior>:/opt/project openmethane-prior python scripts/omDownloadInputs.py
+```
```
Comment (crdanielbusch) [inline README.md:206]: Is this actually relevant for users? Or do you only need docker if you run the application on a server like we do?

File: README.md [hunk 2]
```diff
@@ -36,70 +36,90 @@ The Open Methane prior can be installed from source into a virtual environment w
...
 
+The `Makefile` contains the set of commands to create the virtual environment.
+You can read the instructions out and run the commands by hand if you wish.
+
+### Input Data
+
+To download all the required input files, run:
+
+```console
+make download
```
Comment (crdanielbusch) [inline README.md:47]: I replaced the typed commands with the make commands and added a note that the full commands can be found in Makefile. We can also do it the other way round

File: README.md (hunk 1 above)
Comment (lewisjared) [reply README.md:220]: True. Good point. If we make developer docs then docker usage can go there, but for now assume that you are running locally

**Comment by lewisjared** [comment]
Nice stuff. Can we add in some docs about the input/output directories

**Comment by crdanielbusch** [comment]
> Nice stuff. Can we add in some docs about the input/output directories

Added a basic description of the data directories.
And changed the terminology in the .env file to make it easier to find in the README.

**Review by lewisjared — APPROVED** [review]
Opps I didn't press the approve button again

**Pull request merged**